### PR TITLE
Paginate book

### PR DESCRIPTION
**Pull request by pdkproitf**

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -10,6 +10,1019 @@
  * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
  * file per style scope.
  *
+ *= require bootstrap.min
  *= require_tree .
  *= require_self
  */
+
+/* ^^^^^^^^^^^^^^^^^^^^^^ book ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^*/
```
Comment (yoyocoder) [inline app/assets/stylesheets/application.css]: you should put book's css into `books,scss` file then Rails will import it to application.css by `*= require_tree .`. It is better to handle your css code.

File: app/assets/stylesheets/style.css [hunk 2]
```diff
@@ -0,0 +1,1005 @@
...
+}
+.product-option-shop {
+    margin-top: 15px;
+}
+
+.single-shop-product {margin-bottom: 50px}
+
+.pagination > li > a, .pagination > li > span {
+  color: #1abc9c;
+}
```
Comment (yoyocoder) [inline app/assets/stylesheets/style.css]: config your sublime to add a new line at the end of file automatically

File: app/views/books/index.html.erb [hunk 3]
```diff
@@ -1,45 +1,17 @@
...
-    <% end %>
-  </tbody>
-</table>
-
-<br>
+<div class="single-product-area">
+  <div class="container">
+      <div class="row">
+        <% @books.each do |book| %>
+          <%= render partial: "signle_book", locals: { book: book } if !book.nil? %>
```
Comment (yoyocoder) [inline app/views/books/index.html.erb]: Is  `<%= render partial: "signle_book", locals: { book: book } if book %>` work?

File: app/views/books/index.html.erb [hunk 4]
```diff
@@ -1,45 +1,17 @@
...
+        <% end %>
+      </div>
+      <div class="page_info">
+        <%= page_entries_info @posts %>
+      </div>
+      <%= will_paginate @posts, :style => 'color:blue' %>
```
Comment (yoyocoder) [inline app/views/books/index.html.erb]: Please use the new hash `style style: 'color:blue`

File: app/views/layouts/_header.html.erb [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+                                                <span class="icon-bar"></span>
+                                        </button>
+                                </div>
+                                <div class="navbar-collapse collapse">
+                                        <ul class="nav navbar-nav">
+                                                <li class="active"><a href="index.html">Home</a></li>
+                                                <li><a href="shop.html">Shop page</a></li>
+                                                <li><a href="single-product.html">Single product</a></li>
+                                                <li><a href="cart.html">Cart</a></li>
+                                                <li><a href="checkout.html">Checkout</a></li>
```
Comment (yoyocoder) [inline app/views/layouts/_header.html.erb]: what is `checkout.html`? 
btw, try to use `link_to`.

File: app/views/books/_signle_book.html.erb [hunk 6]
```diff
@@ -0,0 +1,32 @@
+<div class="col-md-3 col-ms-6" style="margin-bottom: 50px">
+  <div class="single-product">
+    <div class="product-f-image">
+      <%= image_tag book.photo, :class => 'my_img' %>
```
Comment (yoyocoder) [inline app/views/books/_signle_book.html.erb]: `class: 'my_img'`

File: app/views/books/_signle_book.html.erb [hunk 7]
```diff
@@ -0,0 +1,32 @@
+<div class="col-md-3 col-ms-6" style="margin-bottom: 50px">
```
Comment (yoyocoder) [inline app/views/books/_signle_book.html.erb]: Avoid to use inline-style. Should move it to css file.

File: app/views/books/index.html.erb (hunk 3 above)
Comment (yoyocoder) [inline app/views/books/index.html.erb]: I wonder how book can be nil in the loop? :'(

**Comment by pdkproitf** [comment]
fixed. and merge to master

**Pull request closed without merging**